### PR TITLE
Only set QPainter::Antialiasing hint once

### PR DESCRIPTION
**Pull request by Florob**
Minor cleanup.
Noticed this because I had local changes getting rid of deprecated Qt API to avoid `-Werror`.

### CODE REVIEW

File: src/connectionMatrix/paintHelper.cpp [hunk 1]
```diff
@@ -196,7 +196,6 @@ QPainterPath buildHeaderArrowPath(QRect const& rect, Qt::Orientation const orien
 void drawCapabilities(QPainter* painter, QRect const& rect, Model::IntersectionData::Type const type, Model::IntersectionData::State const state, Model::IntersectionData::Flags const& flags)
 {
 	painter->setRenderHint(QPainter::Antialiasing);
-	painter->setRenderHint(QPainter::Antialiasing);
```
Comment (christophe-calmejane) [inline src/connectionMatrix/paintHelper.cpp:199]: Huhu, that's what happens when you fix warnings without looking around ;)

File: libs/QtMate/graph/view.cpp [hunk 2]
```diff
@@ -37,7 +37,7 @@ GraphicsView::GraphicsView(QWidget* parent)
 	setResizeAnchor(NoAnchor);
 	setTransformationAnchor(AnchorUnderMouse);
 	setDragMode(QGraphicsView::ScrollHandDrag);
-	setRenderHints(QPainter::Antialiasing | QPainter::TextAntialiasing | QPainter::Antialiasing | QPainter::SmoothPixmapTransform);
```
Comment (christophe-calmejane) [inline libs/QtMate/graph/view.cpp:40]: I didn't write the original code, and I don't really master Qt that much, but why remove the antialiasing?
Is it included in another flag?

Comment (Florob) [reply libs/QtMate/graph/view.cpp:40]: This is not removing, it's de-duplicating. First and third flag are the same.

Comment (christophe-calmejane) [reply libs/QtMate/graph/view.cpp:40]: Damn, never review code at the end of a hard week :p

**Comment by christophe-calmejane** [comment]
Ok merged your commit, though it's still in my private clone. I'll push this week along with other working changes.
Thanks!

**Pull request closed without merging**